roles/kubernetes-apps/ingress_controller/ingress_nginx/templates/clusterrole-ingress-nginx.yml.j2

```diff
@@ -16,15 +16,16 @@ rules:
   - apiGroups: [""]
     resources: ["services"]
     verbs: ["get", "list", "watch"]
-  - apiGroups: ["extensions","networking.k8s.io"]
```

Comment:
I expect this was in place for some beta API compatibility. Did you check the impact of removing the `extensions` apiGroup?

Reply:
The stable API(networking.k8s.io) of ingress has been added since v1.19, and the beta API of ingress has been removed since Kubernetes v1.22 as https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122
On Kubespray, kube_version_min_required is v1.21.0.
So I think it is fine to remove this `extensions`.

Reply:
Thanks @oomichi for the clarification!